tests/integration/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.15)
 # Old integration tests.
 if (CAPSTONE_BUILD_LEGACY_TESTS)
   enable_testing()
-  set(TEST_SOURCES test_skipdata.c test_iter.c)
+  set(TEST_SOURCES test_skipdata.c test_iter.c test_litbase.c)
```

Comment:
This is no legacy test.
You can copy the code but use an `INTEGRATION_TEST_SRC` variable or similar. But they should be build by default.

Also add it as test with `add_test(NAME "integration_litbase" ...)`.